userland/examples/services/ble-env-sense/test/main.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
+#include <string.h>
+
+#include <timer.h>
+#include <ipc.h>
+
+int _svc_num = 0;
+
+char buf[64] __attribute__((aligned(64)));
```

Comment:
Could this just be a `sensor_update_t[64 / sizeof(sensor_update_t)]` instead? That way you would access it directly without a cast below. (Not a big deal though)